src/components/options/providers/contextMenuItems.svelte

```diff
@@ -86,24 +96,40 @@ initData();
           </div>
 
           <div class="flex-1">
-            <div class="d-flex align-items-center">
+            <div class="d-flex align-items-start">
               <div class="flex-1">
                 <input
                   type="text"
                   value="{item.label}"
                   class="form-control text-black"
+                  class:is-invalid="{!item.label &&
+                    !hideErrors[`${index}.label`]}"
                   placeholder="Label to be used in the context menu"
+                  on:input="{() => onInput(index, 'label')}"
+                  on:blur="{() => onBlur(index, 'label')}"
                   on:change="{(e) =>
                     onChange(index, 'label', e.target.value)}" />
+                <div class="invalid-feedback ml-1">
+                  Label should not be empty
+                </div>
               </div>
               <div class="flex-2 mx-2">
                 <input
                   type="text"
                   value="{item.link}"
                   class="form-control text-info"
+                  class:is-invalid="{!isUrl(item.link) &&
+                    !hideErrors[`${index}.link`]}"
                   placeholder="URL address to which send requests"
+                  on:input="{() => onInput(index, 'link')}"
+                  on:blur="{() => onBlur(index, 'link')}"
                   on:change="{(e) =>
                     onChange(index, 'link', e.target.value)}" />
+                <div class="invalid-feedback ml-1">
```

Comment:
Please also note that you shouldn't _render_ the `<div class="invalid-feedback">` element if the input is valid.

![image](https://user-images.githubusercontent.com/8558447/153954799-741513ca-5c8a-4dd5-926c-29d29d8e4efa.png)

> The `<div>` is visually _hidden_ when the form is valid **but** since it's rendered in the DOM, it will be reachable by screen readers applications.